modules/src/what_cocktail.py

```diff
@@ -0,0 +1,351 @@
+#rewrite the program (which a burglar stole from me!) and then release it onto telegram and messenger!
+''' argh what cocktail can i make with these ingredients? search and return a cocktail which has
+the most matching ingredients'''
+from enchant.checker import SpellChecker
+import re
+
+print("What? Cocktail")
+print()
+print()
```

Comment:
Empty print?

Reply:
I inserted these to make the output more readable, I can replace them with print("\n") if you would prefer?

Reply:
Yep, that would be more readable - adding `\n` at the end of the string.

Reply:
done